pkgs/development/compilers/ghc/8.0.1.nix

```diff
@@ -57,6 +74,15 @@ stdenv.mkDerivation rec {
   stripDebugFlags = [ "-S" ] ++ stdenv.lib.optional (!stdenv.isDarwin) "--keep-file-symbols";
 
   postInstall = ''
+    # ghci uses mmap with rwx protection at it implements dynamic
```

Comment:
Indeed, I copied it verbatim from the existing PaX bits applied to 7.6.3... 
Which in turn seem to be based on Gentoo's ebuilds.... which looks to originate with [this commit](https://github.com/gentoo-haskell/gentoo-haskell/commit/13d65c27e1723ad9d2dacd1c4cf6d0b184aebf40) (from 2010).

I'll fix the typo in our tree instead of propagating it 👍